nupd/fetchers/github.py

```diff
@@ -33,18 +47,64 @@ class GHRepository:
     owner: str
     repo: str
     branch: str
-
+    meta: MetaInformation = field(
+        converter=lambda x: MetaInformation(**x)
+        if not isinstance(x, MetaInformation)
+        else x
+    )
     has_submodules: bool | None
     commit: Commit | None = field(
         converter=lambda x: Commit(**x)
         if not isinstance(x, Commit | None)
         else x
     )
-    meta: MetaInformation = field(
-        converter=lambda x: MetaInformation(**x)
-        if not isinstance(x, MetaInformation)
-        else x
-    )
+    latest_release: GitHubRelease | None
```

Comment:
Unfortunately, you have to set this for every sub-object, so it can be properly serialized from JSON

```suggestion
    latest_release: GitHubRelease | None = field(
        converter=lambda x: GitHubRelease(**x)
        if not isinstance(x, GitHubRelease)
        else x
    )
```